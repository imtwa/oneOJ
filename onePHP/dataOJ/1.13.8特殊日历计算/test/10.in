32
9:10:25 9.6.4357
8:57:35 16.10.2617
11:40:45 14.6.4396
14:30:16 16.6.4498
14:55:50 7.1.2926
9:35:14 9.1.3644
6:44:42 28.8.3834
7:18:21 6.5.3124
8:31:22 5.11.4830
7:7:11 29.5.4703
22:18:15 10.2.4724
10:23:11 17.9.2111
10:29:29 17.7.4382
16:57:28 27.9.2691
14:53:32 9.10.2948
23:35:28 18.2.3174
6:1:20 5.0.4222
1:42:45 23.6.2702
11:6:39 3.2.2006
3:3:55 15.11.2358
18:57:53 18.5.2793
20:17:19 0.1.2469
13:52:3 23.4.2787
19:9:51 3.7.2682
16:58:10 23.8.3415
0:14:48 8.2.3276
8:14:26 27.7.3505
13:35:17 29.8.2504
17:6:31 4.10.4227
1:51:42 21.9.4530
4:57:18 7.1.3620
17:28:9 2.11.4855
