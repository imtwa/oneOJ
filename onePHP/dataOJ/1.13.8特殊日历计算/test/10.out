3:82:23 37.1.861
3:73:32 44.7.225
4:86:63 87.3.875
6:4:35 44.6.912
6:22:10 22.3.338
3:99:46 68.5.600
2:81:4 95.1.670
3:4:40 59.7.410
3:55:11 46.10.1033
2:96:65 99.4.987
9:29:34 61.10.994
4:32:76 2.9.40
4:37:14 6.3.870
7:6:57 53.7.252
6:20:50 33.6.346
9:82:96 44.9.428
2:50:92 74.6.811
0:71:35 74.6.256
4:62:95 26.3.2
1:27:71 76.1.131
7:90:19 76.8.289
8:45:35 100.3.171
5:77:81 59.6.287
7:98:50 80.3.249
7:7:6 53.1.517
0:10:27 89.1.466
3:43:35 98.9.549
5:66:16 24.4.184
7:12:85 72.7.813
0:77:56 28.4.924
2:6:45 100.7.591
7:27:88 74.1.1043
